server/android/Cargo.toml

```diff
@@ -9,21 +9,21 @@ path = "src/lib.rs"
 name = "remote_server_android"
 
 [dependencies]
-actix-web = { version = "4.0.1", features = ["rustls"] }
-futures = "0.3"
-jni = { version = "0.19.0" }
+actix-web = { version = "4.7.0", features = ["rustls"] }
+futures = "0.3.30"
+jni = { version = "0.21.1" }
 log = "0.4.14"
-log-panics = { version = "2", features = ["with-backtrace"] }
-once_cell = "1.9.0"
-rcgen = "0.9.2"
+log-panics = { version = "2.1.0", features = ["with-backtrace"] }
+once_cell = "1.19.0"
+rcgen = "0.13.1"
 repository = { path = "../repository" }
 service = { path = "../service" }
 server = { path = "../server", default-features = false, features = [
     "android",
 ] }
-tokio = { version = "1.17.0", features = ["macros"] }
-android_logger = { version = "0.13.0" }
-simple-log = { version = "1.6" }
+tokio = { version = "1.38.0", features = ["macros"] }
```

Comment:
Tokio and other depenancies are used in lots of places, can we move them to be workspace?
E.g. 
`tokio = { workspace = true }`

Then we should mostly only need to update our workspace versions?
Personally I'd try to move as many as possible to this approach. Should help with version consistency across our crates...

Reply:
Can do

Reply:
Thanks!